systemtests/CMakeLists.txt

```
@@ -74,34 +99,38 @@ message(STATUS "POSTGRES_BIN_PATH is ${POSTGRES_BIN_PATH}")
 
 find_program(PERL perl)
 
+set(SYSTEMTESTS_USE_WINE
+    ""
+    CACHE STRING "systemtests do use WINE to run Bareos"
+)
```

If you want a boolean cache variable, you can use option.
```suggestion
option(SYSTEMTESTS_USE_WINE
       "systemtests do use WINE to run Bareos"
       OFF
)
```

---

Applied.